projects/plugins/crm/src/automation/commons/triggers/companies/class-company-created.php

```diff
@@ -10,15 +10,15 @@
 use Automattic\Jetpack\CRM\Automation\Base_Trigger;
 
 /**
- * Adds the Company_New class.
+ * Adds the Company_Created class.
  */
-class Company_New extends Base_Trigger {
+class Company_Created extends Base_Trigger {
 
 	/** Get the slug name of the trigger
 	 * @return string
 	 */
```

Comment:
Just a reminder, we should open an issue to cleanup these comments to comply with WP Standards.